mmv1/third_party/terraform/services/secretmanager/resource_secret_manager_secret_test.go.erb

```diff
@@ -379,6 +379,58 @@ func TestAccSecretManagerSecret_ttlUpdate(t *testing.T) {
 	})
 }
 
+func TestAccSecretManagerSecret_versionDestroyTtlUpdate(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckSecretManagerSecretDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecretManagerSecret_withoutVersionDestroyTtl(context),
+			},
+			{
+				ResourceName:            "google_secret_manager_secret.secret-basic",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"ttl", "labels", "terraform_labels"},
+			},
+			{
+				Config: testAccSecretManagerSecret_basic(context),
```

Comment:
Will this step be needed in between `testAccSecretManagerSecret_withoutVersionDestroyTtl` and `testAccSecretManagerSecret_versionDestroyTtlUpdate` ? That's fine is so, but just checking

Reply:
Removing this condition.